### DISCUSSION TITLE
塔罗牌占卜

### DISCUSSION THREAD

**Discussion by SakuraAzusa** [FloatTech/ZeroBot-Plugin#168] [Ideas]
功能名：塔罗牌占卜
功能具体：每日可随机一张塔罗牌
附带2233娘牌组图链接（里面的大阿卡纳牌）：https://www.aliyundrive.com/s/cvbxLQQ9wD5/folder/61000cc1c78a1da52ef548beb9591a01bdb09a79
解读方面可参考这份文档：https://github.com/AlerHugu3s/PluginVoodoo/blob/master/data/PluginVoodoo/TarotData/Tarots.json

还有一种包括四大元素的完整塔罗牌，可以使用上面链接除额外4张外所有78张牌
塔罗牌详细解读参考文档：https://occultsound.com/learning-thoth-tarot/

**Comment by fumiama** [comment]
这个可以参考百人一首`hyaku`

**Comment by fumiama** [comment]
#213

**Comment by KafCoppelia** [comment]
拿来吧你，我tarot插件更新就靠你这素材了
